x/superfluid/keeper/epoch_test.go

```diff
@@ -63,6 +63,16 @@ func (s *KeeperTestSuite) TestUpdateOsmoEquivalentMultipliers() {
 			// Note: this does not error since CL errors are surrounded in `ApplyFuncIfNoError`
 			expectedZeroMultipler: true,
 		},
+		{
+			name:               "update native token Osmo equivalent successfully",
+			asset:              types.SuperfluidAsset{Denom: "foo", AssetType: types.SuperfluidAssetTypeNative, PricePoolId: 1},
+			expectedMultiplier: osmomath.MustNewDecFromStr("2.0"),
+		},
+		{
+			name:          "update native token Osmo equivalent successfully - no pool",
+			asset:         types.SuperfluidAsset{Denom: "foo", AssetType: types.SuperfluidAssetTypeNative},
+			expectedError: errors.New("failed to get twap price"),
+		},
```

Comment:
I think we do something similar inside transmutter. So, IIUC, the risk here is that twap is wrong or manipulated to increase very rapidly. I feel like twap by definition prevents this, isn't that right?

If it did happen, the risk would be that during one epoch (or as many epochs as the error exist), the native asset being staked would be counted as more value than it is worth. An attacker would have to time the attack so that the slash happens at this point, loosing less real value. I haven't done the math, but my intuition is that this would be very costly.  

In the next PR I untroduce risk factors per asset, which means that we will already be discounting the value of the asset by a percentage, but I think the main question is the one above: is the fact that twap does averages enough to prevent this?

Reply:
oh, actually. The problem might be on the other direction. If the asset crashes, the twap won't be adjusted down sufficiently quickly.

Reply:
Discussed offline. Should be ok